assets/src/scripts/monitoring-location/store/instantaneous-value-time-series-data.js

```diff
@@ -257,7 +259,14 @@ const retrieveExtendedIVTimeSeries = function(siteno, period, paramCd=null) {
         const state = getState();
         const thisParamCd = paramCd ? paramCd : getCurrentParmCd(state);
         const tsRequestKey = getTsRequestKey ('current', period, thisParamCd)(state);
+        const currentUserInputCustomTimeRangeSelectionButton = period !== 'P7D' && period !== 'P30D' && period !== 'P1Y' ? 'custom' : period;
```

Comment:
I wonder if a small helper function for the condition part of the ternary operator would be a good thing. You could also use it in line 265.

Reply:
Oh, neat idea. Will do.